libs/fabric/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -61,8 +64,9 @@ import { ISelectableDroppableTextProps, ISelectableOption } from 'office-ui-fabr
   styles: ['react-renderer'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class FabDropdownComponent extends ReactWrapperComponent<IDropdownProps> implements OnInit {
+export class FabDropdownComponent extends ReactWrapperComponent<IDropdownProps> implements OnInit, AfterContentInit {
   @ViewChild('reactNode') protected reactNodeRef: ElementRef;
+  @ContentChild(DropdownOptionsDirective) readonly dropdownOptionsDirective?: DropdownOptionsDirective;
```

Comment:
Please make sure that the imperative syntax of `options` still works as it did before, in addition to the declarative syntax.

Also, what happens when both syntaxes are used at once?

Reply:
Interesting thought. I actually did test this with combo-box, but not both at the same time, just one at a time. I'll test and get back to you.

But yes, the old style works fine.

Reply:
Tested. Both work in isolation (just-declarative, or just-imperative). If both are used at the same time, the declarative version 'wins'.